pkg/network/networker.go

```diff
@@ -266,10 +267,10 @@ func (n *networker) Leave(networkdID pkg.NetID, containerID string) error {
 // ZDBPrepare sends a macvlan interface into the
 // network namespace of a ZDB container
 func (n networker) ZDBPrepare(hw net.HardwareAddr) (string, error) {
-	netNSName, err := ifaceutil.RandomName("zdb-ns-")
-	if err != nil {
-		return "", err
-	}
+	netNSName := fmt.Sprintf(
```

Comment:
does this naming convention always ensure the namespace name will be smaller then 16 characters ?

Reply:
No, but we can drop few characters from the prefix I think `zdb-` is enough, no. Also I don't believe there is a limitation on the name of the namespace. This works perfectly well on my dev node.

Reply:
@delandtj aren't network namespace name also limited in length like network interface are ?